.vscode/launch.json

```diff
@@ -178,6 +178,22 @@
       "env": {
         "NODE_ENV": "development"
       }
-    }
+    },
+
+    {
```

Comment:
Thanks for adding this!

Actually a question, how to pass in variable for 
```
"args": ["wdio.conf.ts", "--spec", "${file}"],
```
from the debugger?